daemon/nbd-runner.c

```diff
@@ -130,8 +130,8 @@ static void *nbd_map_svc_thread_start(void *arg)
     sin.sin_port = htons(iport);
 
     if (bind(listenfd, (struct sockaddr*)&sin, sizeof(struct sockaddr)) < 0) {
-        nbd_out("bind on port %d failed, %s\n", iport, strerror(errno));
-        nbd_out("will try to use port %d!\n", ++iport);
+        nbd_info("bind on port %d failed, %s\n", iport, strerror(errno));
+        nbd_info("will try to use port %d!\n", ++iport);
```

Comment:
It will be better as WARN messages here.

Reply:
OK